src/azure-cli/azure/cli/command_modules/redis/_help.py

```diff
@@ -188,3 +188,23 @@
 short-summary: Regenerate Redis cache's access keys.
 long-summary: Usage example - az redis regenerate-keys --name testCacheName --resource-group testResourceGroup --key-type {Primary, Secondary}
 """
+
+helps['redis identity'] = """
+type: group
+short-summary: Manage identity assigned to Azure cache for Redis.
+"""
+
+helps['redis identity assign'] = """
+type: command
+short-summary: Assign identity for Azure cache for Redis.
+"""
+
+helps['redis identity remove'] = """
+type: command
+short-summary: Remove the identity already assigned for Azure cache for Redis.
+"""
+
+helps['redis identity show'] = """
+type: command
+short-summary: Show the identity assigned for Azure cache for Redis.
+"""
```

Comment:
```suggestion
"""

```
to make CI linter happy